README.md

```diff
@@ -1,7 +1,57 @@
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+
 # MPC interface
 
 
 This package provides a structure to formulate QP based MPC problems with any linear dynamics, cost functions and constraint.
 All required QP matrices are generated automatically with the structure considered in [qpsolvers](https://scaron.info/doc/qpsolvers/).
 
-This code is mainly based on python, the c++ is a work in progress which does not work yet.
+This repository is mainly based on python, the c++ part is a work in progress which does not work yet.
+
+
+## Dependancies
+
+`mpc_interface` depends on standard python libraries and `qpsolvers` + `osqp` for the examples.
+
+### Standard python packages
```

Comment:
Those are not "standard". By "standard", we mean packages from the Python Standard Library: https://docs.python.org/3/library/index.html
I think you should merge those 2 sections

Reply:
perfect, I am pushing in 5 minuts